_entities.md

```diff
@@ -150,7 +150,7 @@ pair     | The currency pair AB represents moving from A to B.
   "denomination": {
     "amount": "5.00",
     "currency": "GBP",
-    "pair": "GBPUSD",
+    "pair": "GBP-USD",
```

Comment:
This was also not changed.

Reply:
The ticker endpoint accepts both notations, but for more current symbols the old notation becomes ambiguous, so we do not return them. In order to access these newer symbols, it requires usage of the newer notation, with the delimiter.